plugins/org.eclipse.n4js.utils/src/org/eclipse/n4js/utils/io/FileUtils.java

```diff
@@ -157,7 +157,11 @@ public static Path createTempDirectory(String prefix) {
 					"Error while trying to create folder at " + parent + ".");
 		}
 		child.deleteOnExit();
-		return child.toPath();
+		try {
+			return child.getCanonicalFile().toPath();
```

Comment:
imo, canonization is not something this method should do. Most clients won't need it, right?

Reply:
Ok, I move this to AbstractCliCompileTest#setupWorkspace()